### ISSUE TITLE
mismatch between numpy and cunumeric

### ISSUE THREAD

**Issue by rohany** [nv-legate/cunumeric#443]
In the following program, numpy's squeeze and cunumeric's squeeze do different things:

```
import numpy
import cunumeric
import scipy.stats as stats
import scipy.sparse as sparse

class Normal(stats.rv_continuous):
    def _rvs(self, *args,  size=None, random_state=None):
        return random_state.standard_normal(size)

def sample(N: int, D: int, density: float, seend: int):
    NormalType = Normal(seed=seend)
    SeededNormal = NormalType()
    return sparse.random(N, D, density=density, format='csr', dtype=numpy.float64,
                         random_state=seend, data_rvs=SeededNormal.rvs)

seed = 471014
x_cunumeric = cunumeric.array(sample(500, 1, 0.1, seed).todense()).squeeze() # Does not work
x_numpy = cunumeric.array(numpy.array(sample(500, 1, 0.1, seed).todense()).squeeze())
print(x_cunumeric.shape, x_numpy.shape)
```

prints:
```
(1, 500) (500,)
```

**Comment by manopapad** [comment]
@rohany has this been fixed by #448?

**Comment by rohany** [comment]
Yup, closing.
